console/my-account/troubleshooting/cant-log-into-my-account.mdx

```diff
@@ -0,0 +1,53 @@
+---
+meta:
+  title: I cannot log into my account
+  description: This page helps you log into your account
+content:
+  h1: I cannot log into my account
+  paragraph: This page helps you log into your account
+dates:
+  validation: 2022-10-19
+  posted: 2022-10-19
+---
+
+Sometimes you may have problems when connecting to your Scaleway account. If you have encountered any of the following possible cases, check out our solutions.
```

Comment:
```suggestion
Sometimes you may have problems when connecting to your Scaleway account. If you have encountered any of the following possible cases, check out our solutions.
```

Can we change this "Sometimes you may have problems" sounds a bit like there could be issues :)